dna/clutter/clutter.js

```diff
@@ -38,17 +38,150 @@ function unfollow(userAddress){
                   ]});
 }
 
+
 function post(post) {
-    var key = commit("post",post);        // Commits the post block to my source chain, assigns resulting hash to 'key'
+    var post_hash = commit("post",post);        // Commits the post block to my source chain, assigns resulting hash to 'key'
     var me = getMe();                       // Looks up my hash address and assign it to 'me'
                                             // which DHT nodes will use to request validation info from my source chain
+    commit("post_links",{Links:[{Base:me,Link:post_hash,Tag:"post"}]});
+
+   // TODO detect a hash
+    debug(post);
+    debug(post.message);
+    debug("Starting HASHtag search");
+    hashTag=detectHashtag(post.message);
```

Comment:
"detectHashtag()" makes me think it returns just one hashtag but it actually returns an array. maybe rename to "detectHashtagS()" and "hashTag" should be named "hashTagList"